### ISSUE TITLE
need latest resolver..same with google ads and firebase

### ISSUE THREAD

**Issue by dominicdev** [playgameservices/play-games-plugin-for-unity#2264]
my app crash when with the latest plugins google ads admob and firebase (anylitics,authentication and database).. and I cant find any in the log.. only after 
 [Play Games Plugin DLL] 08/10/18 13:13:10 +08:00 DEBUG: Creating Android IPlayGamesClient Client

then icrashed

**Comment by dominicdev** [comment]
when will get update?

**Comment by vaibhavgpt-google** [comment]
Closing this issue as part of a cleanup of older tickets. Google Play Games v1 is now deprecated. Please upgrade to version v2.1.0 or newer: https://github.com/playgameservices/play-games-plugin-for-unity/releases/tag/v2.1.0. If this issue persists on the new v2 architecture, please open a new ticket.
